tutorial/westeros/westeros_baseline.ipynb

```diff
@@ -492,19 +506,23 @@
    "outputs": [],
    "source": [
     "year_df = scenario.vintage_and_active_years()\n",
-    "vintage_years, act_years = year_df['year_vtg'], year_df['year_act']\n",
+    "vintage_years, act_years = year_df[\"year_vtg\"], year_df[\"year_act\"]\n",
     "\n",
+    "# Create the base dataframe for \"output\" and \"input\" parameter\n",
     "base = {\n",
-    "    'node_loc': country,\n",
-    "    'year_vtg': vintage_years,\n",
-    "    'year_act': act_years,\n",
-    "    'mode': 'standard',\n",
-    "    'time': 'year',\n",
-    "    'unit': '-',\n",
+    "    \"node_loc\": country,\n",
+    "    \"year_vtg\": vintage_years,\n",
+    "    \"year_act\": act_years,\n",
+    "    \"mode\": \"standard\",\n",
+    "    \"time\": \"year\",\n",
+    "    \"unit\": \"-\",\n",
     "}\n",
     "\n",
-    "base_input = make_df(base, node_origin=country, time_origin='year')\n",
-    "base_output = make_df(base, node_dest=country, time_dest='year')"
+    "# Extend the base dataframe specific for the \"input\" parameter\n",
+    "base_input = make_df(base, node_origin=country, time_origin=\"year\")\n",
```

Comment:
This is the deprecated signature, right? I'll try to update it to use the preferred one.